avocado/utils/ssh.py

```diff
@@ -248,3 +248,28 @@ def copy_files(self, source, destination, recursive=False):
             return result.exit_status == 0
         except process.CmdError as exc:
             raise NWException("failed to copy file {}".format(exc))
+
+    def check_connection(self):
+        """
+        Check for ssh connection if failed remove control path file
+        and reconnect it.
+
+        :returns: True if success or else False
+        :rtype: bool
+        """
+        if not self.connect():
+            if not self.quit():
+                for root, dirct, files in os.walk("/root/.ssh"):
+                    for file in files:
+                        if file.startswith("avocado-master-root"):
+                            path = os.path.join(root, file)
+                            os.remove(path)
```

Comment:
Another way to remove this is:

```python
file_list = glob.glob('~/.ssh/*avocado-master-root*')
for path in file_list:
   os.remove(path)
```
didn't test the code, just for have an idea.

Reply:
glob.glob('~/.ssh/*avocado-master-root*') with ~/.ssh this not getting any output, so used /root/.ssh.